### PR TITLE
 ex17_ch4_v2.py

### PR DESCRIPTION
**Pull request by amymhaddad**
Create a program that prompts for your weight, gender, number of drinks, the amount of alcohol by volume of the drinks consumed, and the amount of time since your last drink. Calculate your blood alcohol content (BAC) using this formula
BAC = (A × 5.14 / W × r) − .015 × H

Display whether or not it’s legal to drive by comparing the blood alcohol content to 0.08.

### CODE REVIEW

File: ex17_ch4_v2.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+#step1: write out constants
+BAC_ILLEGAL_TO_DRIVE = 0.08
+
+#step2: get input
+gender = input("Enter your gender, male or female: ")
```
Comment (paulghaddad) [inline ex17_ch4_v2.py]: All good variable names on lines 13- 19!

File: ex17_ch4_v2.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+weight = int(input("Enter your weight in pounds: "))
+
+time = int(input("Enter the number of hours since your last drink. If it's been less than one hour enter 1: "))
+
+alcohol = int(input("Enter the amount of alcohol in ounces: "))
+
+#step3: set up two conditionals based on male or female response (since the ratio for the BAC equation depends on gender)
+if "male" in gender:
```
Comment (paulghaddad) [inline ex17_ch4_v2.py]: You're comparing a string to "male", so you should use `if "male" == gender`. The `in` operator is best to check if an item is `in` a collection, such as a list or tuple.  Same applies to line 28.

File: ex17_ch4_v2.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    male_bac = (alcohol * 5.4 / weight * 0.73) - .015 * time
```
Comment (paulghaddad) [inline ex17_ch4_v2.py]: Nice job with this calculation!

File: ex17_ch4_v2.py [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    print("Your BAC is {:.2f}".format(male_bac)+".")
```
Comment (paulghaddad) [inline ex17_ch4_v2.py]: Excellent!!!

File: ex17_ch4_v2.py [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+    if male_bac >= BAC_ILLEGAL_TO_DRIVE:
```
Comment (paulghaddad) [inline ex17_ch4_v2.py]: Excellent!

File: ex17_ch4_v2.py [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+        print("It's not legal for you to drive.")
+
+elif "female" in gender:
+    female_bac = (alcohol * 5.4 / weight * 0.66) - .015 * time
```
Comment (paulghaddad) [inline ex17_ch4_v2.py]: Everything looks good here except changing the `in` to an `==`.

File: ex17_ch4_v2.py (hunk 2 above)
Comment (amymhaddad) [reply ex17_ch4_v2.py]: is it:
"male" == gender
or
gender == "male"?

**Comment by amymhaddad** [comment]
@paulghaddad I made changes to this exercise.

**Pull request closed without merging**